docs/proposed/config-split.md

```diff
@@ -110,6 +110,7 @@ section dedicated to non boolean configuration options anymore.
 ### In `tf_psa_crypto_config.h`, we have:
 #### SECTION Platform abstraction layer
 ```
+//#define MBEDTLS_MEMORY_BUFFER_ALLOC_C
```

Comment:
Rather between MBEDTLS_MEMORY_BACKTRACE and MBEDTLS_MEMORY_DEBUG, alphabetic order.